web/src/app/rotations/RotationDetails.js

```diff
@@ -85,7 +92,12 @@ export default class RotationDetails extends React.PureComponent {
           title={data.rotation.name}
           details={data.rotation.description}
           titleFooter={summary}
-          pageFooter={<RotationUserList rotationID={this.props.rotationID} />}
+          pageFooter={
+            <RotationUsersList
+              rotationID={this.props.rotationID}
+              rotation={data.rotation}
```

Comment:
Shouldn't this be handled by the nested component?

Reply:
From what we discussed last week, we were to pass that data down from the parent so things would load together

Reply:
Also, it made converting to hooks much easier, so I went with it